webpack/build.config.js

```diff
@@ -0,0 +1,31 @@
+var srcFolder = './src';
```

Comment:
Could we please `git mv`, so it is easier to follow changes (and go back in history, if needed)?